crates/rethnet_eth/src/remote/client.rs

```diff
@@ -214,7 +214,7 @@ impl RpcClient {
     }
 
     /// Submit a consolidated batch of RPC method invocations in order to obtain the set of data
-    /// contained in AccountInfo.
+    /// contained in [`AccountInfo`].
```

Comment:
fyi ["Backticks around the link will be stripped"](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html) so i don't _think_ they have any effect here

Reply:
There is actually a difference:

![image](https://github.com/NomicFoundation/hardhat/assets/6917585/9639db6c-4aed-4966-b52d-38383d7f152d)